otterdog/eclipse-kuksa.jsonnet

```diff
@@ -80,18 +91,7 @@ orgs.newOrg('eclipse-kuksa') {
       gh_pages_source_path: "/",
       web_commit_signoff_required: false,
       branch_protection_rules: [
-        orgs.newBranchProtectionRule('main') {
-          required_approving_review_count: 0,
-          requires_linear_history: true,
-          requires_status_checks: false,
-          requires_strict_status_checks: true,
-        },
-        orgs.newBranchProtectionRule('master') {
-          required_approving_review_count: 0,
-          requires_linear_history: true,
-          requires_status_checks: false,
-          requires_strict_status_checks: true,
-        },
+        kuksa_default_branch_protection_rule('master')
```

Comment:
We do not have "main" branch so no reason to protect it, right?

Reply:
Probably. For the way website is set up, @lukasmittag has the most up-to date knowledge, but it is a bit special anyway